modular_skyrat/modules/modular_weapons/code/automatic.dm

```diff
@@ -51,7 +51,9 @@
 	fire_sound = 'sound/weapons/gun/smg/shot_alt.ogg'
 	weapon_weight = WEAPON_MEDIUM
 	w_class = WEIGHT_CLASS_BULKY
-	company_flag = COMPANY_CANTALAN
+
+/obj/item/gun/ballistic/automatic/cfa_wildcat/give_manufacturer_examine()
+	AddComponent(/datum/component/manufacturer_examine, COMPANY_CANTALAN)
 
 /obj/item/gun/ballistic/automatic/cfa_wildcat/no_mag
```

Comment:
(ditto) also supposed to be automatic as a baseline, with burst as the secondary, and if you're getting rid of one, it should default to full auto